src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -103,47 +146,54 @@ export async function estimateGas(
 export async function calculateDelta(
   panopticish: Panoptic,
   req: CalculateDeltaRequest
-): Promise<any> {
+): Promise<CalculateDeltaResponse> {
   const result = await panopticish.calculateDelta(req.PRICE, req.RANGE, req.STRIKE);
-  return result;
+  return {
+    delta: result
+  };
 }
 
 export async function calculateGamma(
   panopticish: Panoptic,
   req: CalculateGammaRequest
-): Promise<any> {
+): Promise<CalculateGammaResponse> {
   const result = await panopticish.calculateGamma(req.PRICE, req.RANGE, req.STRIKE);
-  return result;
+  return {
+    gamma: result
+  };
 }
 
-
 // Subgraph interactions
 
 export async function queryOpenPositions(
```

Comment:
Should we rename this `queryPositions` since it returns both closed and open positions? And rename associated types as well, then?

Reply:
Done: 9b8d2254c3cfe138da8b155d6429ed946a910f58